src/TwigComponent/doc/index.rst

```diff
@@ -840,6 +840,117 @@ The ``with`` data is what's mounted on the component object.
 .. note::
 
     Embedded components *cannot* currently be used with LiveComponents.
+    
+Component HTML Syntax
+-----------------
+    
+.. versionadded:: 2.9
+
+    This syntax was been introduced in 2.8 and is still experimental: it may change in the future.
+    
+Twig Component came with a new syntax to ease the readability of your template:
+
+.. code-block:: html+twig
+
+    {# templates/some_page.html.twig #}
+    <twig:Alert warning :user="user.username" message="I am an alert!">
+       <div class="alert-content">
+            // ... your component content
+       </div>
+       <twig:block name="footer">
+            // ... footer content
+       </twig:block>
+    </twig:Alert>
+    
+You can now use a custom tag prefix starting by <twig: and your component name
+
+.. code-block:: html+twig
+
+    <twig:Alert></:Alert>
+    // and also self closing tag
+    <twig:Alert/>
+    
+You can pass an attribute to your component like any HTML attribute
```

Comment:
```suggestion
You can pass props to your component by using HTML attributes. Suppose you have the following component:
```

Then show a simple `Alert` component with the `warning`, `message` and `user` properties (including any others I missed). I think it helps to see the component so you can see how the attributes "bind" to them